homeassistant/components/hassio/backup.py

```diff
@@ -114,28 +114,30 @@ def __init__(self, hass: HomeAssistant) -> None:
     async def async_create_backup(
         self,
         *,
-        addons_included: list[str] | None,
         agent_ids: list[str],
-        database_included: bool,
         backup_name: str,
-        folders_included: list[str] | None,
+        include_addons: list[str] | None,
+        include_all_addons: bool,
```

Comment:
We dont use `include_all_addons` in the body. Shouldn't we do that?

Reply:
Supervisor doesn't yet support it

Reply:
Are we going to update before the release? Otherwise we may need a way to tell the frontend about the missing support.

Reply:
We need a supervisor release with new features, not just this one.

Reply:
I'd add a comment about our plans to avoid confusion among readers, even if it's for just a couple of days.